jetty-openid/src/main/java/org/eclipse/jetty/security/openid/OpenIdAuthenticator.java

```diff
@@ -366,6 +371,17 @@ public void prepareRequest(ServletRequest request)
         baseRequest.setMethod(method);
     }
 
+    private boolean hasExpiredIdToken(HttpSession session)
+    {
+        if (session != null)
+        {
+            Map<String, Object> claims = (Map)session.getAttribute(CLAIMS);
```

Comment:
This is guarenteed to be safe? Should you check in the condition of the if statement on line 376 that the attribute  is a map? e.g.
`if (session != null && session.getAttribute(CLAIMS) instanceof Map)`